ui/src/actions/authors.js

```diff
@@ -172,13 +172,18 @@ export function assignDifferentProfileClaimedPapers({ from, to }) {
         'publicationSelectionCanNotClaim'
       );
       assigning();
-      await http.post('/assign/author', {
+      const { data } = await http.post('/assign/author', {
         from_author_recid: from,
         to_author_recid: to,
         papers_ids_already_claimed: claimedPapers,
         papers_ids_not_matching_name: unclaimablePapers,
       });
-      assignSuccessDifferentProfileClaimedPapers();
+      if ('created_rt_ticket' in data) {
```

Comment:
it's safer to use `data.hasOwnProperty('created_rt_ticket')` because `in` operator will search for everything including prototype.

Reply:
🤢